addons/armaos/functions/fnc_terminal_removeChar.sqf

```diff
@@ -0,0 +1,26 @@
+params ["_computer"];
+
+private _terminal = _computer getVariable "AE3_terminal";
+
+private _terminalBuffer = _terminal get "AE3_terminalBuffer";
+private _terminalPrompt = _terminal get "AE3_terminalPrompt";
+private _terminalCursorPosition = _terminal get "AE3_terminalCursorPosition";
+
+private _lastBufferLineIndex = (count _terminalBuffer) - 1;
+private _lastBufferLine = _terminalBuffer # (_lastBufferLineIndex);
+
+private _terminalMaxRows = 25;
+private _terminalMaxColumns = 50;
+
+if ((count _lastBufferLine) > (count _terminalPrompt)) then
+{
+	_terminalCursorPosition = _terminalCursorPosition - 1;
+	_lastBufferLine = _lastBufferLine select [0, (count _lastBufferLine) - 1];
+};
+
+_terminalBuffer set [_lastBufferLineIndex, _lastBufferLine];
+
+_terminal set ["AE3_terminalBuffer", _terminalBuffer];
+_terminal set ["AE3_terminalCursorPosition", _terminalCursorPosition];
+
+_computer setVariable ["AE3_terminal", _terminal];
```

Comment:
Should be unnecessary, because variables referencing hashmaps are only pointers to the actual hashmap object, not the hashmap itself.

Reply:
removed in https://github.com/y0014984/Advanced-Equipment/pull/90/commits/996717f1cedd397770d94ec876aedb11e65687ad